tests/draft-next/additionalProperties.json

```diff
@@ -152,5 +152,29 @@
                 "valid": true
             }
         ]
+    },
+    {
+        "description": "additionalProperties with propertyNames",
+        "schema": {
+            "$schema": "https://json-schema.org/draft/next/schema",
+            "propertyNames": {
+                "maxLength": 5
+            },
+            "additionalProperties": {
+                "type": "number"
+            }
+        },
+        "tests": [
+            {
+                "description": "Valid against both keywords",
+                "data": { "apple": 4 },
+                "valid": true
+            },
+            {
+                "description": "Valid against propertyNames, but not unevaluatedProperties",
```

Comment:
this says `unevaluatedProperties`, should say `additionalProperties` I think.
same for the other two.

Reply:
Yeah! let me just correct it and raise the pull request again.
I will have to be more careful...